packages/react-native-editor/package.json

```diff
@@ -55,23 +55,23 @@
 		"jsdom-jscore-rn": "git+https://github.com/iamcco/jsdom-jscore-rn.git#a562f3d57c27c13e5bfc8cf82d496e69a3ba2800",
 		"node-fetch": "^2.6.0",
 		"react-native": "0.64.0",
-		"react-native-gesture-handler": "git+https://github.com/wordpress-mobile/react-native-gesture-handler.git#1.10.1-wp-3",
-		"react-native-get-random-values": "git+https://github.com/wordpress-mobile/react-native-get-random-values.git#v1.4.0-wp-1",
+		"react-native-gesture-handler": "https://raw.githubusercontent.com/wordpress-mobile/react-native-gesture-handler/e20fb879b029dc2da92ff06e059cc8b1bd1cee26/react-native-gesture-handler-1.10.1-wp-3.tgz",
+		"react-native-get-random-values": "https://raw.githubusercontent.com/wordpress-mobile/react-native-get-random-values/f3f5e2a04078e4b00f6a3ca38ac1c73be942a2c9/react-native-get-random-values-1.4.0-wp-1.tgz",
 		"react-native-hr": "git+https://github.com/Riglerr/react-native-hr.git#2d01a5cf77212d100e8b99e0310cce5234f977b3",
-		"react-native-hsv-color-picker": "git+https://github.com/wordpress-mobile/react-native-hsv-color-picker#v1.0.1-wp-1",
-		"react-native-keyboard-aware-scroll-view": "git+https://github.com/wordpress-mobile/react-native-keyboard-aware-scroll-view.git#v0.8.8-wp",
-		"react-native-linear-gradient": "git+https://github.com/wordpress-mobile/react-native-linear-gradient.git#v2.5.6-wp-1",
+		"react-native-hsv-color-picker": "https://raw.githubusercontent.com/wordpress-mobile/react-native-hsv-color-picker/03c0038419326fb86120560cdc3ce958d52cee37/react-native-hsv-color-picker-1.0.1-wp-1.tgz",
+		"react-native-keyboard-aware-scroll-view": "https://raw.githubusercontent.com/wordpress-mobile/react-native-keyboard-aware-scroll-view/v0.8.8-wp-1/react-native-keyboard-aware-scroll-view-0.8.8-wp-1.tgz",
```

Comment:
This is the only dependency that uses tag instead of commit hash in the URL. In the future, when we introduce changes in a forked repository, we would need to generate a tarball and a tag so it will be expected that tarball URLs have the tag, as we have for `react-native-keyboard-aware-scroll-view` package.

I'm not sure why we didn't generate a new tag for the rest of the repositories but probably it was related to the fact that we're not introducing changes and only the tarball file.